src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -41,10 +45,23 @@ public CommandResult execute(Model model, OfficeConnectModel officeConnectModel)
         }
 
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+        Id pId = personToDelete.getId();
+        List<Id> taskIdList = getTaskIdList(officeConnectModel, pId);
+
+        RepositoryModelManager<AssignTask> assignTaskModelManager = officeConnectModel.getAssignTaskModelManager();
+        taskIdList.forEach(tId -> assignTaskModelManager.deleteItem(new AssignTask(pId, tId)));
```

Comment:
might violate law of demeter. perhaps a method could be created in officeConnectModel that allows deleting of items from assignTaskModelManager?